Version 2.0

Tranz5 opened the pull request:
Changes for more stable block chain.
2min block time
1 day old min age
Old Clients will be disconnted on Friday Sep 15th

Tranz5 committed:
Version 2.0

Changes for more stable block chain.
2min block time
1 day old min age
Old Clients will be disconnted on Friday Sep 15th

Tranz5 committed:
Update README.md

Tranz5 commented:
Let me know if there are any issues. Or please pull in as soon as you are able to.

Thanks!

ahmedbodi commented on src/main.cpp:
```diff
@@ -84,6 +82,23 @@ int64_t nSplitThreshold = GetProofOfWorkReward();
 
+unsigned int SetTargetSpacing()
+{
+   unsigned int nStakeTargetSpacingSwitch = (fTestNet ? 1 * 60 : 1 * 30); // block spacing, 1 min tesnet, 30 seconds Main (OLD)
```
Wouldn't GetTargetSpacing be a better name?

ahmedbodi commented on src/main.cpp:
```diff
@@ -84,6 +82,23 @@ int64_t nSplitThreshold = GetProofOfWorkReward();
+unsigned int SetStakeMinAge()
+{
+   unsigned int nStakeMinAgeSwitch = (fTestNet ? 2 * 60 * 60 : 60 * 60 * 24 * 10); // minimum age for coin age - 2hrs TestNet, 10 days Main (OLD)
+   if (nBestHeight + 1 > (fTestNet ? SPACING_TARGET_SWITCH_TESTNET : SPACING_TARGET_SWITCH))
```
Same as above

ahmedbodi commented:
aside from the SetTarget functions which i think would be better called GetTarget it all looks good to me.

Tranz5 commented:
Thanks ahmed. I am going to update a few other things here in a few days I will put that on the list..

Tranz5 committed:
2.0.1.0

Added SplitBlock to GUI. @presstab Hyperstake
Increaed default splitthreshold and combinethreshold, plus updated Max
for each. This will work in startup and config switchs up to 2500/5000.
And further increases in gui.
Changed max percentage for Stake For Charity to 100%
Renamed SetStakeMinAge to GetStakeMinAge
Renamed SetTargetSpacing to GetTargetSpacing
Created stub function for GetModiferInterval. Will be used to fork and
allow for 1 day min stake period.
Fixed CoinControl Weight to correct show at 8.8 days until fork fixes
it.
Fixed Stake Esimate in GUI.
Fixed Fee Esitmate when using CoinControl.
Bumped version to 2.0.1.0

Tranz5 commented:
Alright I have updated to 2.0.1.0. Should take care of those renames and added a few other features and fixes.
